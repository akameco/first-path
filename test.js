// @flow
const path = require('path')
const firstPath = require('.')

test('throw errors', () => {
  // $FlowFixMe
  expect(() => firstPath(1)).toThrowErrorMatchingSnapshot()
})

test('return first path', () => {
  const file = path.join('src', 'nest', 'file')
  expect(firstPath(file)).toBe('src')
})

test('return first path with string', () => {
  const file = 'nest/nest1/nest2/nest3'
  expect(firstPath(file)).toBe('nest')
})
