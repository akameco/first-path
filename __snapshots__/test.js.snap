// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`throw errors 1`] = `"Expected a string, got number"`;
